api4/system.go

```diff
@@ -903,6 +937,16 @@ func completeOnboarding(c *Context, w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	firstAdminCompleteSetupObj := model.System{
+		Name:  model.SystemFirstAdminCompleteSetup,
+		Value: "true",
+	}
+
+	if err := c.App.Srv().Store.System().SaveOrUpdate(&firstAdminCompleteSetupObj); err != nil {
+		c.Err = model.NewAppError("setFirstAdminCompleteSetup", "api.error_set_first_admin_complete_setup", nil, err.Error(), http.StatusInternalServerError)
+		return
+	}
```

Comment:
Let's keep all the business logic in the `app` layer and move code into `app.CompleteOnboarding`